src/plugins/restrict/restrict.ts

```diff
@@ -41,7 +39,9 @@ const restrict = (
             filename: name,
             restriction,
             chunkNames: usageAssets,
-            issuerPath: issuerPath?.map((ip) => ip.name),
+            // davidhouweling: for some reason I couldn't cast it to "Module['issuerPath']"
```

Comment:
Maybe this could be a TODO comment so that it gets highlighted?

Reply:
Well it isn't really a task that needs doing? I just left a comment as to why I did this the way I did...